chef/cookbooks/crowbar/recipes/crowbar-db-dump.rb

```diff
@@ -106,6 +110,10 @@
       done
     EOF
   end
+  # Deletes postgres data dir in non-HA scenario
+  directory "/var/lib/pgsql/data" do
+    action :delete
```

Comment:
If you do have serious concerns, I'm fine removing the directory. It's really a question of whether we can afford another safety net; it's not strictly required.